### PR TITLE
Send email events also the the new analytical platform

### PR DESCRIPTION
**Pull request by ddippolito**
## Trello ticket URL
https://trello.com/c/Wp0OMdII/310-tv-migration-blockers

## Changes in this PR:

When migrating to the new Analytical Platform we forgot to add mail events to the migrations. This is a temporary PR to make sure we can send events correctly. And will refine at a later stage

### CODE REVIEW

File: config/analytics_custom_events.yml [hunk 1]
```diff
@@ -18,6 +19,9 @@ shared:
   - jobseeker_application_submitted
   - jobseeker_subscription_update
   - jobseeker_confirmation_instructions
+  - jobseeker_inactive_account
+  - jobseeker_account_closed
+  - jobseeker_job_listing_ended_early
```
Comment (scruti) [inline config/analytics_custom_events.yml:24]: Question: :raising_hand_man: 
Are these new events we want to send to send? 
Just trying to figure out if this is strictly related to the email events or just using the PR to add some extra info on top of triggering email events.

**Review by scruti — APPROVED** [review]

File: db/seeds.rb [hunk 2]
```diff
@@ -17,6 +17,7 @@
 users = [
   { email: "alex.lee@education.gov.uk", family_name: "Alex", given_name: "Lee" },
   { email: "alisa.ali@education.gov.uk", family_name: "Ali", given_name: "Alisa" },
+  { email: "brandon.chan@education.gov.uk", family_name: "Chan", given_name: "Brandon" },
```
Comment (scruti) [inline db/seeds.rb:20]: I think Brandon email is: `brandon1.chan@education.gov.uk`. It shows like this on Slack, Outlook profile...

File: app/controllers/subscriptions_controller.rb [hunk 3]
```diff
@@ -86,7 +86,7 @@ def notify_new_subscription(subscription)
 
   def trigger_create_job_alert_clicked_event
     request_event.trigger(:vacancy_create_job_alert_clicked, vacancy_id: StringAnonymiser.new(vacancy_id))
-    trigger_dfe_analytics_event(:vacancy_create_job_alert_clicked, { vacancy_id: StringAnonymiser.new(vacancy_id) })
+    trigger_dfe_analytics_event(:vacancy_create_job_alert_clicked, { vacancy_id: StringAnonymiser.new(vacancy_id).to_s })
```
Comment (scruti) [inline app/controllers/subscriptions_controller.rb:89]: Is the Vacancy ID a PII?  We are removing the anonymiser calls on it further down the PR.

File: db/seeds.rb (hunk 2 above)
Comment (brandonchan1) [reply db/seeds.rb:20]: Correct, I have brandon1.chan@education.gov.uk

File: app/controllers/subscriptions_controller.rb (hunk 3 above)
Comment (brandonchan1) [reply app/controllers/subscriptions_controller.rb:89]: Agree with @scruti. It's just id so does not need to be anonymised

**Review by scruti — APPROVED** [review]
Re-checked and LGTM!

**Pull request merged**